examples/Test Sequencer/generic dut/sequencer_generic_dut.py

```diff
@@ -0,0 +1,136 @@
+import os
+import sys
+import time
+import csv
+from flexlogger.automation import Application, FlexLoggerError, TestSessionState
+
+
+def test_sequence1(config_file_path):
+    """Launch FlexLogger, reference the CSV file,
+    open the specified FlexLogger projects,
+    and control a simulated DUT.
+
+    The CSV lists the path to the FlexLogger projects,
+    DUT commands, and test times.
+
+    To use this example with your DUT or other external hardware,
+    replace the 'dut_control' function with your specific DUT's API
+    or communication methods, as needed.
+    """
+    print("\nLaunching FlexLogger...")
+    with Application.launch() as app:
+        print("FlexLogger Launched Successfully!\n")
+        dut_control("powerOn")
+        dut_control("updateFirmware")
+        test1(app, config_file_path)
+        dut_control("powerOff")
+    print("FlexLogger Closed Successfully")
+    return
+
+
+def test1(app_reference, config_path):
+    with open(config_path, newline="") as csvfile:
+        csvdata = csv.reader(csvfile)
+        next(csvdata)  # skip header
```

Comment:
[nit/optional/suggestion] Instead of just always skipping the first line because it's the header, I'd have some syntax for a "comment" line. Specifically, if the line starts with "#" (or "//" or something) you could ignore it. That would let the user document what each testpoint might be testing.

But these are just examples, so maybe that's overkill

Reply:
I think that is a great idea, but think that's probably overkill for the example. That could be an option if we update these in a future release.